python/clx/analytics/cybert.py

```diff
@@ -0,0 +1,227 @@
+import cupy
+import numpy as np
+import os
+import pandas as pd
+import torch
+import torch.nn.functional as F
+import logging
+from collections import defaultdict
+from torch.utils.dlpack import from_dlpack
+from transformers import BertForTokenClassification
+
+log = logging.getLogger(__name__)
+
+
+class Cybert:
+    """
+    Cyber log parsing using BERT. This class provides methods for
+    loading models, prediction, and postprocessing.
+    """
+
+    def __init__(self):
+        self._model = None
+        self._label_map = {}
+        self._vocab_lookup = {}
+        self._vocabpath = self.__get_vocab_file_path()
```

Comment:
Since `__get_vocab_file_path` and `__get_hash_table_path` are so similar, small and only referenced here, something like this might be a little bit more concise.
```
resources_dir="%s/resources" % os.path.dirname(os.path.realpath(__file__))
self._vocabpath= "%s/bert-base-cased-vocab.txt" % resources_dir
self._hashpath = "%s/bert-base-cased-hash.txt" % resources_dir
```
Let me know if that works ok?